### DISCUSSION TITLE
Upgrade to 9.8.1 and Telerik removal

### DISCUSSION THREAD

**Discussion by dnnsmuggler** [dnnsoftware/Dnn.Platform#4469] [General] [answered]
Hi!

When upgrading DNN from version before 9.8.0 direct to 9.8.1, can we then use the same process to remove Telerik as described in the release notes for version 9.8.0?

Or do we first need to upgrade to version 9.8.0 and remove Telerik as described in the release notes for version 9.8.0?

Is it correct that Telerik is NOT removed automatically when updating to version 9.8.0 or 9.8.1, but will be automatically removed in the coming version 10?

/Peter

**Comment by sleupold** [comment, marked as answer]
AFAIK, Telerik removal procedure is identical for DNN 9.8.0 and 9.8.1 and the upgrade between does not affect the removal.

**Comment by dnnsmuggler** [comment]
Thanks! I will check it out.
